src/sentry_plugins/github/endpoints/webhook.py

```diff
@@ -51,6 +56,24 @@ def __call__(self, organization, event):
                 author_email = u'{}@localhost'.format(
                     author_email[:65],
                 )
+            # try to figure out who anonymous emails are
+            elif '@users.noreply.github.com' in author_email:
```

Comment:
I'd explicitly do:

```python
elif author_email[-25:] == '@users.noreply.github.com'
```

And work strictly with the assumption that we end with this exact string. This allows avoiding splitting on `@` too. Can then just do:

```python
gh_username = author_email[:-25]
```

This will make it a bit less error prone.